staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -165,6 +165,25 @@ __kubectl_get_resource()
     __kubectl_parse_get "${nouns[${#nouns[@]} -1]}"
 }
 
+__kubectl_get_resources()
+{
```

Comment:
For portability, do you mind to set a variable with name like separator=','  ? Not sure if in the future we change people change mind and use a different separator like "#", "foobar".

Reply:
done